### ISSUE TITLE
Warning for view functions

### ISSUE THREAD

**Issue by Li-ReDBox** [PyCQA/pylint-django#155]
Django's view functions have signature of [`request`](https://docs.djangoproject.com/en/2.0/topics/http/views/). Even I don't use `reqest`, I would like to include it as part of signature. But if I run:

```shell
pylint --load-plugins pylint_django views.py

W:184,12: Unused argument 'request' (unused-argument)
```

My question is: is there a way to surpass this warning?

**Comment by atodorov** [comment]
Right now you can disable with `# pylint: disable=unused-argument` or rename to `_request`. 

It should be possible to teach pylint_django to ignore this error but I'll have to see how to do that. May not be very easy though.

**Comment by Li-ReDBox** [comment]
Thanks @atodorov for replying. Change the argument to `_request` does not look like a bad idea. How pylint treats some functions with signature of `func(arg1, arg2,  *args, **kwargs)` which is the common pattern of Django's view class methods?
